flowzone.yml

```diff
@@ -3135,7 +3173,9 @@ jobs:
 
       - *deleteDraftGitHubRelease
       - *checkoutVersionedSha
-      - *getReleaseNotes
+      - <<: *getReleaseNotes
+        env:
+          user_defined: ${{ needs.release_notes.outputs.body }}
```

Comment:
I know you added `look_back="${look_back:-1}"` but I think providing both required envs here would be cleaner, and we don't have to look at the function to figure out the default value.

Reply:
Done